data/youtube

```diff
@@ -171,4 +176,5 @@ youtubei.googleapis.com
 youtubekids.com
 youtubemobilesupport.com
 yt.be
+yt3.ggpht.com
```

Comment:
Duplicated with https://github.com/v2fly/domain-list-community/pull/756/files#diff-f8917947d128b748ff2ecf053d69193b84de69d30fd0778ca95562ed70f03ea0R5

Reply:
ok, i removed it.